namui-cli/src/procedures/release_wasm_electron.rs

```diff
@@ -0,0 +1,96 @@
+use crate::{
+    debug_println,
+    services::{
+        electron_package_service::{Arch, ElectronPackageService, Platform},
+        resource_collect_service::{CollectOperation, ResourceCollectService},
+        rust_build_service::{BuildOption, BuildPlatform, BuildResult, RustBuildService},
+    },
+    util::{
+        get_cli_root_path, get_namui_config, overwrite_hot_reload_script_with_empty_file,
+        print_build_result,
+    },
+};
+use std::{
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+
+pub fn release_wasm_electron(
+    manifest_path: &Path,
+    platform: Option<Platform>,
+    arch: Option<Arch>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let build_dist_path = manifest_path.parent().unwrap().join("pkg");
+    let namui_static_path = get_cli_root_path().join("www");
+    let project_root_path = manifest_path.parent().unwrap().to_path_buf();
+
+    let electron_package_service = ElectronPackageService::new();
+    let package_result = electron_package_service.package_electron_app(arch, platform)?;
+
+    let release_path = project_root_path.join("target").join("namui").join(format!(
+        "release_wasm_electron_{}_{}",
+        &package_result.platform, &package_result.arch
+    ));
+
+    let rust_build_service = Arc::new(RustBuildService::new());
+    let resource_collect_service = ResourceCollectService::new(&project_root_path, &release_path);
+
+    build(
+        rust_build_service.clone(),
+        build_dist_path.clone(),
+        project_root_path.clone(),
+    )?;
+
+    let namui_config = get_namui_config(&project_root_path)?;
+    let mut ops: Vec<CollectOperation> = namui_config
+        .resources
+        .iter()
+        .map(|src_dest_path_pair| {
+            CollectOperation::new(
+                &PathBuf::from(&src_dest_path_pair.0),
+                &PathBuf::from("resources").join(&src_dest_path_pair.1),
+            )
+        })
+        .collect();
+    ops.push(CollectOperation::new(
+        &namui_static_path,
+        &PathBuf::from("resources"),
+    ));
+    ops.push(CollectOperation::new(
+        &build_dist_path.join("bundle.js"),
+        &PathBuf::from("resources"),
+    ));
+    ops.push(CollectOperation::new(
+        &build_dist_path.join("bundle_bg.wasm"),
+        &PathBuf::from("resources"),
+    ));
+    ops.push(CollectOperation::new(
+        &PathBuf::from(&package_result.output_path),
+        &PathBuf::from(""),
+    ));
+    resource_collect_service.collect_resources(ops)?;
+
+    overwrite_hot_reload_script_with_empty_file(&release_path)?;
```

Comment:
In release, there is no hotReload server.
So overwirte hotReload.js to not to try connect to hotReload server.